apps/src/dance/song-selector.module.scss

```diff
@@ -0,0 +1,33 @@
+@import "color";
+
+.songSelectorWrapper {
+
+  .selectStyle {
+    width: calc(100% - 42px);
```

Comment:
nit: oh sorry I didn't catch this before - but might be nice to have some constants for these values for clarity (ex. some `$preview-button-size: 32px, $default-margin: 10px` so that this can be `calc(100% - $preview-button-size - $default-margin)`)

Reply:
Thanks for spotting this! Fixed